priv/repo/migrations/20240621085408_add_struct_name_to_beacon_component_attrs.exs

```diff
@@ -0,0 +1,9 @@
+defmodule Beacon.Repo.Migrations.AddStructNameToBeaconComponentAttrs do
+  use Ecto.Migration
+
+  def change do
+    alter table(:beacon_component_attrs) do
+      add :struct_name, :string
```

Comment:
Having this field really helped setting up the form in the UI - with this a lot of code was cut 🔪  - also, I think that is more clear to have the field `type` just to save the `component_type` and this field specific to save the **struct name** if the `type` is `struct`